teuthology/orchestra/daemon/cephadmunit.py

```diff
@@ -4,14 +4,14 @@
 
 log = logging.getLogger(__name__)
 
-class CephDaemonUnit(DaemonState):
+class CephadmUnit(DaemonState):
```

Comment:
nit: CephAdmUnit?

Reply:
i went with CephadmUnit because 'cephadm' is one word (not 'ceph-adm').